plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/topk.cpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+
+#include "common_test_utils/test_constants.hpp"
+#include "single_layer_tests/topk.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+    InferenceEngine::Precision::FP32,
+    InferenceEngine::Precision::FP16,
+};
+
+const std::vector<int64_t> axes = {
+    0,
+    1,
+    2,
+};
+
+const std::vector<int64_t> k = {
+    1,
+    5,
+    10,
+};
+
+const std::vector<ngraph::opset4::TopK::Mode> modes = {
+    ngraph::opset4::TopK::Mode::MIN,
+    ngraph::opset4::TopK::Mode::MAX,
+};
+
+const std::vector<ngraph::opset4::TopK::SortType> sortTypes = {
+    // ngraph::opset4::TopK::SortType::NONE,
+    ngraph::opset4::TopK::SortType::SORT_INDICES,
+    ngraph::opset4::TopK::SortType::SORT_VALUES,
+};
+
+INSTANTIATE_TEST_CASE_P(smoke_TopK, TopKLayerTest,
```

Comment:
How long does the test take? Usually we add a separate smoke test which has only one or very few tests.

Reply:
For now without `ngraph::opset4::TopK::SortType::NONE` there are 72 cases. I copied this from the other plugin and I will add a seperate `smoke` test instance.

Reply:
Thanks, please remove the smoke_ prefix of this test as well.